route53_crossfeed_app.tf

```diff
@@ -347,16 +375,6 @@ resource "aws_route53_record" "crossfeed_staging_api_CNAME2" {
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
-resource "aws_route53_record" "crossfeed_staging_api_CNAME3" {
-  provider = aws.route53resourcechange
-
-  name    = "staging.api.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
```

Comment:
staging.api was also a typo, it's supposed to be api.staging -- just changed in this commit.